shared/src/business/entities/cases/Case.ts

```diff
@@ -808,7 +808,7 @@ export class Case extends JoiValidationEntity {
   }
 
   hasPrivatePractitioners() {
-    return this.privatePractitioners.length > 0;
+    return this.privatePractitioners!.length > 0;
```

Comment:
is this correct? 

It would seem that `privatePractitioners` is optional, and so we might have to check for its existence rather than assume it exists. Why not a `?` instead of a `!`?